internal/service/encryptionatrest/resource_encryption_at_rest.go

```diff
@@ -215,15 +215,9 @@ func (r *encryptionAtRestRS) Create(ctx context.Context, req resource.CreateRequ
 
 	projectID := encryptionAtRestPlan.ProjectID.ValueString()
 	encryptionAtRestReq := &admin.EncryptionAtRest{}
-	if encryptionAtRestPlan.AwsKmsConfig != nil {
-		encryptionAtRestReq.AwsKms = NewAtlasAwsKms(encryptionAtRestPlan.AwsKmsConfig)
-	}
-	if encryptionAtRestPlan.AzureKeyVaultConfig != nil {
-		encryptionAtRestReq.AzureKeyVault = NewAtlasAzureKeyVault(encryptionAtRestPlan.AzureKeyVaultConfig)
-	}
-	if encryptionAtRestPlan.GoogleCloudKmsConfig != nil {
-		encryptionAtRestReq.GoogleCloudKms = NewAtlasGcpKms(encryptionAtRestPlan.GoogleCloudKmsConfig)
-	}
+	encryptionAtRestReq.AwsKms = NewAtlasAwsKms(encryptionAtRestPlan.AwsKmsConfig)
+	encryptionAtRestReq.AzureKeyVault = NewAtlasAzureKeyVault(encryptionAtRestPlan.AzureKeyVaultConfig)
+	encryptionAtRestReq.GoogleCloudKms = NewAtlasGcpKms(encryptionAtRestPlan.GoogleCloudKmsConfig)
```

Comment:
knitpit: so you can do:

	encryptionAtRestReq := &admin.EncryptionAtRest{
AwsKms: NewAtlasAwsKms(encryptionAtRestPlan.AwsKmsConfig),
AzureKeyVault: NewAtlasAzureKeyVault(encryptionAtRestPlan.AzureKeyVaultConfig),
GoogleCloudKms: NewAtlasGcpKms(encryptionAtRestPlan.GoogleCloudKmsConfig),
}